docs/INSTALL-devel-flowchart.txt

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
We had decided that the change from haas to HIL will be made in a dedicated PR.
If it was not decided so, I would have done the changes before hand. 

This is unnecessary work that is lingering the PR.